### PR TITLE
Sidebar open-close indicator

### PR DESCRIPTION
**Pull request by AntonioMateoGomez**
This PR adds edit icon in doc section´s searchbar, sidebar open-close indicator.
Also includes some minor style modification, and fixes some links adding `relative_url`.

### CODE REVIEW

File: _includes/_doc-wrapper.html [hunk 1]
```diff
@@ -7,6 +7,11 @@
           <img src="{{ '/img/docs/search-icon.svg' | relative_url }}" title="search" alt="Search">
           <input id="docsearch" class="search" placeholder="Search documentation..." type="search">
         </div>
+        <a target="_blank" class="edit-icon"
```
Comment (calvellido) [inline _includes/_doc-wrapper.html]: I wouldn't add this yet, as the doc edition might not be possible with the new structure

File: _sass/components/docs/_doc-header.scss [hunk 2]
```diff
@@ -36,6 +36,12 @@
       input[type="search"]::-webkit-search-results-decoration { display: none; }
     }
 
+    .edit-icon {
```
Comment (calvellido) [inline _sass/components/docs/_doc-header.scss]: Same here

File: _includes/_doc-wrapper.html (hunk 1 above)
Comment (AntonioMateoGomez) [reply _includes/_doc-wrapper.html]: ok, we can add it easyly later

File: _sass/components/docs/_doc-header.scss (hunk 2 above)
Comment (AntonioMateoGomez) [reply _sass/components/docs/_doc-header.scss]: ok, we can add it easyly later

**Pull request merged**